feat(tianmu): Improve the readbility of stonedb. (#11)

konghaiya opened the pull request:
[summary]
1 class member variable refactor：exproter directory; 
2 class member variable refactor：loader directory;

<!--

Thank you for contributing to StoneDB!

PR Title Format: <type>(<scope>): description to this pr (#issue_id)
e.g.
fix(util): fix sth..... (#1)

-->

## Summary about this PR
<!--

There MUST be one line starting with "Issue Number:  " and 
linking the relevant issues via the "close" or "ref".
e.g.:
Issue: close #1

-->

Issue Number: close #issue_number_you_created


## Tests Check List
<!-- At least one of next options must be included. -->

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog
<!-- At least one of next options must be included. -->

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation
<!-- At least one of next options must be included. -->

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

hustjieke reviewed (approved):
LGTM

RingsC commented on storage/tianmu/loader/load_parser.cpp:
```diff
@@ -29,44 +29,44 @@ namespace Tianmu {
-      no_obj(attrs[0]->NumOfObj()) {
+    : attrs_(attrs),
+      start_time_(types::RCDateTime::GetCurrent().GetInt64()),
+      ioparam_(iop),
```
as the style of `start_time` and `pack_size`.  suggest: `io_param_`. and `no_obj_` changes to `num_of_obj_`

konghaiya replied:
The modification has been completed

RingsC commented on storage/tianmu/loader/load_parser.h:
```diff
@@ -49,32 +49,32 @@ class LoadParser final {
+  uint pack_size_ = 0;
+  Rejecter rejecter_;
+  uint cur_row_ = 0;
+  int64_t no_obj_ = 0;
```
Here, using `no_obj` to `num_of_obj_`.

Do you know the meaning of `row_no_` and `dup_no_` ? what are they used for?

RingsC commented on storage/tianmu/loader/parsing_strategy.h:
```diff
@@ -37,28 +37,28 @@ class ParsingStrategy final {
-  CHARSET_INFO *cs_info;
-  std::vector<char> temp_buf;
+  std::vector<core::AttributeTypeInfo> attr_infos_;
+  bool prepared_;
```
Overall rules: using the unify naming style and meanful names.

RingsC reviewed (approved):
LGTM

konghaiya committed:
feat(tianmu): Improve the readbility of stonedb. (#11)

[summary]
1 class member variable refactor：exproter directory;
2 class member variable refactor：loader directory;
